saleor/graphql/channel/mutations/channel_update.py

```diff
@@ -146,6 +120,11 @@ def check_permissions(cls, context, permissions=None, **data):
         if "order_settings" in input and len(input) == 1:
             permissions.append(OrderPermissions.MANAGE_ORDERS)
 
+        # Permission MANAGE_CHECKOUTS or MANAGE_CHANNELS is required to update
+        # checkoutSettings.
+        if "checkout_settings" in input and len(input) == 1:
```

Comment:
I'm wondering why we are checking the number of elements in the `input` 🤔 ?

Reply:
Took it from the lines above. In case o updating only checkout settings the input len will be 1. In that case we can allow  to call the mutation with checkout_permissions

Reply:
We should also allow users with `MANAGE_ORDERS` and `MANAGE_CHECKOUTS` updates `order_settings` and `checkout_settings` together.